robots.txt

```diff
@@ -0,0 +1,2 @@
+User-Agent: *
+Disallow: /
```

Comment:
Similar thing to head noindex - this will prevent robots crawling production as well....